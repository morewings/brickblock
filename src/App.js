import React from 'react';
import { Container, Row, Col } from 'reactstrap';
import Tabs from 'components/Tabs/Tabs';

const App = props => (
  <div className="App">
    <Container>
      <Row>
        <Col xs="12">
          <Tabs />
        </Col>
      </Row>
    </Container>
  </div>
);

export default App;
