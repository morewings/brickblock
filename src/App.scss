@import "variables.scss";

.contributors {
  background: $primaryBg;
  padding: $marginVertical $marginVertical;
  border-radius: $borderRadius;
}