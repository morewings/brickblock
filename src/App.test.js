import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import App from './App';

const getData = jest.fn();

const defaultProps = {
  getData,
};

describe('App', () => {
  beforeEach(() => {
    getData.mockClear();
  });
  it('renders without crashing', () => {
    const wrapper = shallow(<App {...defaultProps} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  it('calls `this.props.getData` after mount', () => {
    shallow(<App {...defaultProps} />);
    expect(getData).toHaveBeenCalledTimes(1);
  });
});
