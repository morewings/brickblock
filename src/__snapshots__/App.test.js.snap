// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing 1`] = `
<div
  className="App"
>
  <Container
    tag="div"
  >
    <Row
      tag="div"
    >
      <Col
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
        xs="12"
      >
        <Tabs />
      </Col>
    </Row>
  </Container>
</div>
`;
